src/vma/proto/dst_entry.h

```diff
@@ -65,7 +65,7 @@ class dst_entry : public cache_observer, public tostr, public neigh_observer
 
 	virtual void 	notify_cb();
 
-	virtual bool 	prepare_to_send(bool skip_rules=false);
+	virtual bool 	prepare_to_send(bool skip_rules=false, bool is_connect=false);
```

Comment:
I've revised this again.
In principal `is_connect` shouldn't be exposed to `dst_entry_udp` and `dst_entry_udp_mc` just as neither it nor `skip_rules` should be exposed to UDP in `prepare_to_send()`. In all these cases UDP always passes `false`.
The inheritance of `prepare_to_send()` and `resolve_net_dev()` should be fixed up together in a separate commit.